pandablocks/commands.py

```diff
@@ -283,6 +293,23 @@ class GetPcapBitsLabels(Command):
         GetPcapBitsLabels() -> PcapBitsLabels()
     """
 
+    def execute(self) -> ExchangeGenerator[Dict[str, List[str]]]:
+        ex = Exchange("PCAP.*?\n")
```

Comment:
```suggestion
        ex = Exchange("PCAP.*?")
```
Exhanges don't need a newline on the strings to send